api/lib/infrastructure/repositories/user-recommended-training-repository.js

```diff
@@ -8,4 +9,18 @@ module.exports = {
     const knexConn = domainTransaction?.knexTransaction || knex;
     return knexConn(TABLE_NAME).insert({ userId, trainingId, campaignParticipationId });
   },
+
+  async findByCampaignParticipationId({ campaignParticipationId, locale, domainTransaction }) {
```

Comment:
```suggestion
  async findByCampaignParticipationId({ campaignParticipationId, locale, domainTransaction = DomainTransaction.emptyTransaction() }) {
```

Pas nécessaire de faire comme dans la méthode `save` ?

Reply:
Si en effet thanks